.env.example

```
@@ -51,7 +51,22 @@ STRIPE_PUBLISHABLE_KEY=pk_test_...
 STRIPE_WEBHOOK_SECRET=whsec_...
 
 # =============================================================================
-# 3D Asset Generation - Tripo3D
+# 3D Asset Generation - HuggingFace (PRIMARY - Recommended for quality)
+# Get your token from: https://huggingface.co/settings/tokens
+# Create a token with "Read" access for inference API
+# =============================================================================
+HUGGINGFACE_API_TOKEN=hf_...
+HF_HOME=./cache/huggingface
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HF_HOME key should go before the HUGGINGFACE_API_TOKEN key

(UnorderedKey)